querydsl-docs/src/main/docbook/en-US/content/general/creating-queries.xml

```diff
@@ -230,8 +230,8 @@ QBankAccount bankAccount = account.as(QBankAccount.class);
     <para>Literals can be selected by referring to them via Constant expressions. Here is a simple example </para>
 
     <programlisting language="java"><![CDATA[
-query.list(Expressions.constant(1),
-           Expressions.constant("abc"));
+query.select(Expressions.constant(1),
+             Expressions.constant("abc"));
```

Comment:
Did you intentionally omit fetch?
(It's not necessary to have it, since it's only talking about the select part, but just wondering).